investors/urls.py

```diff
@@ -1,9 +1,27 @@
 from rest_framework.routers import DefaultRouter
 from investors.views import InvestorViewSet, SavedStartupViewSet
+from startups.views.startup import StartupViewSet
+
+from django.urls import path
+from .views import (
+    ViewedStartupListView,
+    ViewedStartupCreateView,
+    ViewedStartupClearView
+)
 
 # Register viewsets with the router
 router = DefaultRouter()
 router.register(r'saved', SavedStartupViewSet, basename='saved-startup')
 router.register(r'', InvestorViewSet, basename='investor')
 
-urlpatterns = router.urls
+custom_urls = [
+    # Endpoint 1: GET recently viewed startups
+    path('startups/viewed/', ViewedStartupListView.as_view(), name='viewed-startup-list'),
+
+    # Endpoint 2: POST log a startup view
+    path('startups/view/<int:startup_id>/', ViewedStartupCreateView.as_view(), name='viewed-startup-create'),
```

Comment:
The endpoint path includes <int:startup_id>, but if Startup uses UUIDField as primary key, this will fail. Change <int:startup_id> to <uuid:startup_id> if using UUIDs.
```
path('startups/view/<uuid:startup_id>/', ViewedStartupCreateView.as_view(), name='viewed-startup-create'),
```